src/main/webapp/profile.js

```diff
@@ -24,13 +24,22 @@ function onProfileLoad() {
 function submitProfileForm(methodType, redirectUrl) {
   if(!validateProfileForm()) {
     return;
-  } 
+  }
+  document.getElementById('company-field').placeholder = '';
+  document.getElementById('linkedin-field').placeholder = '';
+  let qualifiedJobs = [];
+  for (let job of possibleQualifiedJobs()) {
+    if (job.checked) {
+      qualifiedJobs.push(job.getAttribute('data-enum-name'));
+    }
+  }
   const personJson = {
     firstName: $('#first-name-field').val(),
     lastName: $('#last-name-field').val(),
     company: $('#company-field').val(),
     job: $('#job-field').val(),
-    linkedin: $('#linkedin-field').val()
+    linkedin: $('#linkedin-field').val(),
```

Comment:
l'm really confused as to why company and linkedIn are null sometimes. linkedin might be because in PersonRequest, the i is capital. l'm happy to do the debugging myself, but am curious as to what you discovered.

Reply:
I was about to look into this. I think you're right and it is just the i being capitalized or not. If that's all then I'll remove all my null checks.

Reply:
Fixed it. It was the i vs I.